lib/snowflexid/protocol.ex

```diff
@@ -49,10 +51,9 @@ defmodule SnowflexId.IdHelper do
     end
   end
 
-  defp new_id(node_id, seq_num) do
-    ts = :os.system_time(:millisecond) - @custom_epoch
+  defp new_id(node_id, seq_num, opts \\ []) do
+    ts = :os.system_time(:millisecond) - Keyword.get(opts, :epoch, @elixir_epoch)
```

Comment:
As this is a very sensitve configuration, better avoid optional parameter and keep it in config file.